src/classes/Handsontable.php

```diff
@@ -0,0 +1,830 @@
+     * @see https://docs.handsontable.com/latest/Options.html#autoWrapRow
+     */
+    protected $autoWrapRow = true;
+
+    /**
+     *
+     * @see https://docs.handsontable.com/latest/Options.html#minSpareRows
+     */
+    protected $minSpareRows = 1;
+
+    /**
+     *
+     * @see https://docs.handsontable.com/latest/Options.html#maxCols
+     */
+    protected $maxCols;
+
+    /**
+     *
+     * @see https://docs.handsontable.com/latest/Options.html#rowHeights
+     */
+    protected $rowHeights;
+
+    /**
+     *
+     * @see https://docs.handsontable.com/latest/Options.html#colWidths
+     */
+    protected $colWidths;
+
+    /**
+     *
+     * @see https://docs.handsontable.com/latest/Options.html#sortIndicator
+     */
+    protected $sortIndicator = false;
+
+    /**
+     *
+     * @see https://docs.handsontable.com/latest/Options.html#manualColumnResize
+     */
+    protected $manualColumnResize = false;
+
+    /**
+     *
+     * @see https://docs.handsontable.com/latest/Options.html#manualRowResize
+     */
+    protected $manualRowResize = false;
+
+    public function __construct($container_name = null, $cellConfig = null)
+    {
+        if ($cellConfig != null) { // load cell configuration
+            $this->setCell($cellConfig);
+        }
+        if (is_null($container_name)) { // create handsontable instance if it doesn't exist
+            $this->containerName = 'handsontable' . Handsontable::$table_created;
+            Handsontable::$table_created++;
+        } else {
+            $this->containerName = $container_name;
+        }
+    }
+
+    public function getContainerName()
+    {
+        return $this->containerName;
+    }
+
+    public function getLoadLibrairy()
+    {
+        return $this->loadLibrairy;
+    }
+
+    public function getLoadDataSource()
+    {
+        return $this->loadDataSource;
+    }
+
+    public function getSaveDataSource()
+    {
+        return $this->saveDataSource;
+    }
+
+    public function getSave()
+    {
+        return $this->save;
+    }
+
+    public function getAutosave()
+    {
+        return $this->autosave;
+    }
+
+    public function getLoadAction()
+    {
+        return $this->loadAction;
+    }
+
+    public function getSaveElementId()
+    {
+        if (is_null($this->saveElementId)) {
+            $this->saveElementId = 'save' . Handsontable::$table_created;
+        }
+        return $this->saveElementId;
+    }
+
+    public function getStartRows()
+    {
+        return $this->startRows;
+    }
+
+    public function getStartCols()
+    {
+        return $this->startCols;
+    }
+
+    public function getWidth()
+    {
+        return $this->width;
+    }
+
+    public function getHeight()
+    {
+        return $this->height;
+    }
+
+    public function getRowHeaders()
+    {
+        return $this->rowHeaders;
+    }
+
+    public function getColHeaders()
+    {
+        return $this->colHeaders;
+    }
+
+    public function getContextMenu()
```

Comment:
in this getter, the method return has differents types. You should documentate the method to explain why